### ISSUE TITLE
[BUG] Improve Lighthouse Accessibility and SEO

### ISSUE THREAD

**Issue by rahulaarora** [UBA-GCOEN/StichHub#133]
## Description

**Describe the bug:**
Images used in the website do not have the ALT attribute and multiple elements have the same ID attribute.

**Expected behaviour:**
The Lighthouse results should have 100 in Accessibility and SEO

**Actual behaviour:**
The Lighthouse results are below 100.

## Reproduction Steps

**Steps to reproduce the behaviour:**

1. Go to https://stichhub-chi.vercel.app/
2. Open Chrome Dev Tools 
3. Go to the Lighthouse tab and click Analyze Page Load

## Screenshots

<img width="474" alt="Screenshot 2023-05-26 at 1 37 47 PM" src="https://github.com/UBA-GCOEN/StichHub/assets/24794534/3c703c66-8f37-473d-887c-a2f964adc1ec">

<img width="892" alt="Screenshot 2023-05-26 at 1 43 56 PM" src="https://github.com/UBA-GCOEN/StichHub/assets/24794534/82cda4dd-7e5c-46c9-a454-b04b6b7cf61e">

<img width="892" alt="Screenshot 2023-05-26 at 1 44 17 PM" src="https://github.com/UBA-GCOEN/StichHub/assets/24794534/9244d1d4-210b-414b-8abf-333639ef72d4">

## Additional Information

**Device (please complete the following information):**
- OS: macOS
- Browser Chrome
- Version latest

**Additional context:**

- [x] There is no similar existing open issue
- [x] This is a part of GSSOC'23

**Comment by rahulaarora** [comment]
@Naresh-chandanbatve I would like to work on this.
I solved this bug in the previous issue #70 but it is showing again

**Comment by Siddhant-Patil0203** [comment]
Okay @rahulaarora 
Go ahead
